iotileship/iotile/ship/recipe_action.py

```diff
@@ -0,0 +1,25 @@
+from __future__ import (unicode_literals, print_function, absolute_import)
+from builtins import str
+
+from iotile.core.exceptions import *
+
+class RecipeActionObject (object):
```

Comment:
I think this entire file can be removed.  As discussed, recipe actions won't actually inherit from this class since they will be  defined in various places so I don't think this class will ever actually be used.

Reply:
yes, i agree